docs/examples/example_ngraph_utils.cpp

```diff
@@ -60,7 +60,7 @@ std::shared_ptr<ngraph::Function> create_advanced_function() {
 }
 // ! [ngraph_utils:advanced_function]
 
-void pattern_matcher_examples() {
+void pattern_matcher_examples(std::shared_ptr<Node> node) {
```

Comment:
Why node is not constant? I didn't see any code which can change this pointer.

Also should we add reference to shared_ptr<Node>?

Reply:
This function is used only to gather code in a single place and will be used only for docs generation.